lib/extensions.js

```diff
@@ -44,6 +44,11 @@ function getHumanArchitecture(arch) {
     case 'ia32': return '32-bit';
     case 'x86': return '32-bit';
     case 'x64': return '64-bit';
+    case 'armv7l': return '32-bit, ARM';
+    case 'armv7hf': return '32-bit, ARM';
+    case 'armv8': return '64-bit, ARM';
+    case 'aarch64': return '64-bit, ARM';
+    case 'arm64': return '64-bit, ARM';
```

Comment:
Looks like drone wont give us full ARM coverage. See https://docs.drone.io/reference/agent/drone-runner-arch/

Reply:
Here, arm means 32 bit and arm64 means 64 bit.
So, it gives full arm coverage.

Reply:
Do you know what the distinction is between armv7l, armv7hf, aarch64? I'm not sure how they map to drone.io availability

Reply:
So it looks like there's a whole bunch of issues with arm detection. I'm not sure detecting the right binary is every possible.

https://github.com/nodejs/node/issues/13629
https://github.com/nodejs/node/issues/9491
https://github.com/nodejs/build/issues/897

Reply:
Hi @xzyfer 

I can understand your problem, it could be a bit tricky to provide support for ARM

As there are 2 categories of ARM machine on the basis of their architecture,
architectures like arm,``` armv6, armv7, armv7l, armv7hf ```are 32 bit and 
```arm64, aarch64, armv8, armv8hf ```are 64 bit

Releasing the binary for both type, 32 bit and 64 bit will cover all the architectures.

Please, let me know if any more information is required.

Reply:
Hi,

So there is a bit of a mix going on here, some of these are node process.arch (arm and arm64) and some of them appear to be the platform descriptions (for example as returned by linux uname).

So armv7l is little endian arm, armv7hf is hard floating point. 

OTOH, its gets a bit more confusing for armv8 as that name tends to be used by linux 32-bit compatibility containers (aka running 32-bit armv7 binaries on aarch64/64 bit machine).